src/SharedFrameworkValidation/RestoreSDKProject/RestoreSDKProject.csproj

```diff
@@ -61,4 +61,14 @@
     <Copy SourceFiles="%(ReferencePath.Identity)" DestinationFolder="$(_MetaPackageDestinationFolder)" />
   </Target>
 
+  <Target Name="CopyNETCoreAppVersionsFile" DependsOnTargets="ResolveLockFileCopyLocalProjectDeps" BeforeTargets="CoreCompile">
+    <PropertyGroup>
+        <NETCoreAppVersionsFilePath Condition="$([System.String]::new('%(FileDefinitions.Identity)').Contains('Microsoft.NETCore.App.versions.txt'))">%(FileDefinitions.ResolvedPath)</NETCoreAppVersionsFilePath>
```

Comment:
> Won't this be potentially ambiguous if there is more then one version?

yes, it's possible that we find two, the one in the identity package and the rid specific one but either will work since corefx version is in both.

> I assume you know the version and the package folder root why can't you create a direct file path?

I don't know the package folder root, this is a SDK project that is being built using `dotnet restore` and `dotnet build` which means that the package will be restored to the `%USERPROFILE%\.nuget\packages` in the default case, but it could also be in a different place if somebody overrides the default restore path, which is why I chose to instead use the calculated one in the restored project which we know will always be correct.